src/utils/index.js

```diff
@@ -12,3 +11,41 @@ export const JWTStorage = {
 		localStorage.removeItem(SESSION_TOKEN);
 	},
 };
+
+export const validateObject = (obj) => {
+	let result = {};
+
+	const isValidPhone = (str) => {
+		const phonePattern = /09\d{9}$/;
+		return phonePattern.test(str);
+	};
+
+	const isStrongPassword = (str) => {
+		const passwordPattern = /^(?=(.*[A-Z])(?=.*?[a-z])(?=(.*[0-9]){3,})(?=.*?[#?!@$%^&*-])).{10,}$/;
+		return passwordPattern.test(str);
+	};
+
+	const isEmpty = (str) => {
+		return str == '';
```

Comment:
specify the variable name. do not use str

Reply:
done